pkgs/development/python-modules/pandas/default.nix

```diff
@@ -251,9 +254,6 @@ buildPythonPackage rec {
   ];
 
   meta = with lib; {
-    # https://github.com/pandas-dev/pandas/issues/14866
-    # pandas devs are no longer testing i686 so safer to assume it's broken
-    broken = stdenv.isi686;
```

Comment:
I think this is nice information to retain. Could you please comment out the `broken`, and rephase the comment to `pandas devs no longer test i686, it's commonly broken`

Reply:
Done!